features/admin.userstores.v1/api/use-get-user-store-details.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import useRequest, {
+    RequestConfigInterface,
+    RequestErrorInterface,
+    RequestResultInterface,
+    RevalidatorInterface,
+    RevalidatorOptionsInterface
+} from "@wso2is/admin.core.v1/hooks/use-request";
+import { store } from "@wso2is/admin.core.v1/store";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosResponse } from "axios";
+import { useState } from "react";
+import { BareFetcher, PublicConfiguration } from "swr/dist/_internal";
+import { UserStoreDetails } from "../models";
+
+/**
+ * Hook to get User Store details.
+ *
+ * @param shouldFetch - If true, will fetch the data.
```

Comment:
Shall we add the `userStoreId` param as well here

Reply:
Fixed 70f200f14255f037a46096205acd8f666919bb76